README.rst

```diff
@@ -19,8 +19,34 @@ Zoo Game
     Some species of animals like eating other species of animals.  For example, lions like eating wildebeest.  If you put prey and predator in the same cage, then all the prey should be eaten by the predator.  (The program should tell you which predator ate which prey.)
 
 
-
-
 Bonus task
 ----------
 2.  Each cage needs a reference number so the zoo keeper doesn't get confused.  Automatically generate a reference number when building each cage.  The first cage should be 1, the second 2, etc.
+
```

Comment:
Note that there are still a lot of binary files (the *.pyc files, etc.) that are still in version control - you should definitely remove these.